ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -59,9 +59,27 @@ public struct ActivityIndicator: View, TokenizedControlView {
                 "Accessibility.ActivityIndicator.Stopped.label".localized
         }()
 
-        SemiRing(color: color,
-                 thickness: tokenSet[.thickness].float,
-                 accessibilityLabel: accessibilityLabel)
+#if DEBUG
+        let accessibilityIdentifier: String = {
+            let status: String = state.isAnimating ? "in progress" : "progress halted"
+            return "Activity Indicator that is \(status) and size \(state.size.rawValue)"
+        }()
+#endif
+
+        let semiRing = {
+#if DEBUG
+            return SemiRing(color: color,
```

Comment:
Optional but I think you might be able to move lines 63-66 above this line?

Reply:
Or (and this is super optional) just define semiring twice, once in the debug ifdef and once not in debug

Reply:
Personally I prefer less if defs, I find them difficult to read but it's just a preference